core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -3263,20 +3309,25 @@ components:
             This hash is `ACC_{N+1} = Blake2b-256(CONCAT(ACC_N, LEDGER_HASH_{N}))`, starting with `ACC_0 = 000..000` the pre-genesis accumulator.
         status:
           $ref: "#/components/schemas/LtsCommittedTransactionStatus"
-        fungible_outcome:
-          description: A list containing all account fungible balance updates
+        fungible_entity_balance_changes:
+          description: A list containing all account fungible sbalance updates
           type: array
           items:
-            $ref: "#/components/schemas/LtsAccountFungibleBalanceChanges"
-    LtsAccountFungibleBalanceChanges:
+            $ref: "#/components/schemas/LtsEntityFungibleBalanceChanges"
+        fee:
+          type: string
+          description: |
+            The string-encoded decimal representing the total amount of XRD payed as fee (execution, validator tip and royalties).
+            A decimal is formed of some signed integer `m` of attos (`10^(-18)`) units, where `-2^(256 - 1) <= m < 2^(256 - 1)`.
+    LtsEntityFungibleBalanceChanges:
       type: object
       required:
-        - account_address
+        - address
```

Comment:
For consistency with other parts of the API, I might suggest calling it `entity_address`